packages/dcos-history/build

```diff
@@ -3,12 +3,18 @@ source /opt/mesosphere/environment.export
 export LIB_INSTALL_DIR="$PKG_PATH/lib/python3.4/site-packages"
 mkdir -p "$LIB_INSTALL_DIR"
 
+
 # Copy from the ro /pkg/extra to a rw folder.
 cp -r /pkg/extra /build
 
+
+# Copy the gunicorn conf file to $PKG_PATH
+envsubst '$PKG_PATH' < /pkg/extra/history/dcos_history_conf.py > "$PKG_PATH/dcos_history_conf.py"
```

Comment:
I'm confused about this usage of envsubst. What is the purpose of '$PKG_PATH'? envsubst does not take an argument other than STDIN I thought...
Furthermore, you are envsubst-ing on an python file...
(1) That python file has no shell variables to substitute
(2) If it did that would be a rather odd python file
Why not just cp the file as this is effectively doing?